sdk/spring/azure-spring-boot-2-2-starter-cosmosdb/README.md

```diff
@@ -0,0 +1,251 @@
+# Azure Cosmos DB Spring Boot 2.2 Starter client library for Java
+
+[Azure Cosmos DB](https://azure.microsoft.com/services/cosmos-db/) is a globally-distributed database service that allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Graph, and Azure Table storage. 
+
+[Package (Maven)][package] | [API reference documentation][refdocs] | [Product documentation][docs] | [Samples][sample]
+
+## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven](http://maven.apache.org/) 3.0 and above
+
+### Include the package
+[//]: # ({x-version-update-start;com.microsoft.azure:azure-cosmosdb-spring-boot-starter;current})
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-cosmosdb-spring-boot-starter</artifactId>
+    <version>2.4.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Key concepts
+- Spring Data ReactiveCrudRepository basic CRUD functionality
```

Comment:
How about we link users to the `Key concepts` section of `azure-spring-boot-2-3-starter-cosmosdb/README.md`, in which case we don't need to maintain two copies of the same content.